docs/core/build/distribution-packaging.md

```diff
@@ -107,13 +107,15 @@ The following lists the recommended packages:
   - **Version:** \<runtime version>
   - **Example:** dotnet-sdk-2.1
   - **Contains:** (3),(4)
-  - **Dependencies:** `aspnetcore-runtime-[major].[minor]`, `dotnet-targeting-pack-[major].[minor]`, `aspnetcore-targeting-pack-[major].[minor]`, `netstandard-targeting-pack-[netstandard_major].[netstandard_minor]`, `dotnet-apphost-pack-[major].[minor]`, `dotnet-templates-[major].[minor]`
+  - **Dependencies:** `dotnet-runtime-[major].[minor]`, `aspnetcore-runtime-[major].[minor]`, `dotnet-targeting-pack-[major].[minor]`, `aspnetcore-targeting-pack-[major].[minor]`, `netstandard-targeting-pack-[netstandard_major].[netstandard_minor]`, `dotnet-apphost-pack-[major].[minor]`, `dotnet-templates-[major].[minor]`
```

Comment:
I'd be curious to learn whether it's generally good to rely on transitive dependencies or not. I don't have an objection, though.

Reply:
@omajid ?

Reply:
I don't think it's an exact science. If it works, it works :smile: 

It seems to me like the SDK depends on `shared/Microsoft.NETCore.App` itself, so a dependency on the runtime package makes sense. But it doesn't need to be mandatory.